src/OrleansRuntime/Placement/IPlacementDirector.cs

```diff
@@ -0,0 +1,25 @@
+using System.Threading.Tasks;
+
+namespace Orleans.Runtime.Placement
+{
+    /// <summary>
+    /// Interface for placement directors.
+    /// </summary>
+    internal interface IPlacementDirector
+    {
+        Task<PlacementResult> OnSelectActivation(
```

Comment:
This is one of the first places I'd like to see us use `ValueTask<T>` when it's available, but I didn't drop the suffix - it was never there :)